lib/utils/types.js

```diff
@@ -21,5 +21,12 @@ module.exports = {
   RENDERER: {
     SIMPLE: 'SIMPLE',
     ROTATING: 'ROTATING'
+  },
+
+  MARKER_TYPES: {
```

Comment:
yeah should add a comment.  I used lower case here to have nicer event names,  if you're writing javascript directly. 

robot.on('red marker assigned',  () => {})